### ISSUE TITLE
Is there any way to include negative images to enhance the accuracy

### ISSUE THREAD

**Issue by hkhaledmohamed** [experiencor/keras-yolo2#401]
Hello Sir, Thanks so much for your effort. please is there any way to add negative samples which do not contain the desired object to enhance the prediction. Is that possible ?. Is that will enhance the accuracy.

**Comment by MBoaretto25** [comment]
@hkhaledmohamed Yes, you can do that as is suggested in[here](https://github.com/experiencor/keras-yolo2/issues/246) for the racoon problem. 
The model must learn what is and what is not the object that it will be set to predict.
